src/main/target/system_stm32f30x.c

```diff
@@ -331,7 +331,13 @@ void SetSysClock(void)
 
     /* PLL configuration */
     RCC->CFGR &= (uint32_t)((uint32_t)~(RCC_CFGR_PLLSRC | RCC_CFGR_PLLXTPRE | RCC_CFGR_PLLMULL));
-    RCC->CFGR |= (uint32_t)(RCC_CFGR_PLLSRC_PREDIV1 | RCC_CFGR_PLLXTPRE_PREDIV1 | RCC_CFGR_PLLMULL9);
+
+    if (HSE_VALUE == 12000000) {
```

Comment:
that's a value of XTAL oscillator. That board has 12MHz while the rest one F3s has 8MHz